bulk-import/ndjson-generator/Helpers/DataGenerator.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace NDJsonGenerator.Helpers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text.Json;
+    using Azure.DigitalTwins.Core.Serialization;
+
+    public class DataGenerator
+    {
+        private static readonly string ModelsTemplate = "{{\"@id\":\"dtmi:com:microsoft:azure:iot:model{0};1\",\"@type\":\"Interface\",\"contents\":[{{\"@type\":\"Property\",\"name\":\"prop0\",\"schema\":\"string\"}},{{\"@type\": \"Component\",\"name\":\"Component1\",\"schema\":\"dtmi:com:microsoft:azure:iot:componentmodel;1\"}},{{\"@type\":\"Property\",\"name\":\"prop1\",\"schema\":\"string\"}},{{\"@type\":\"Relationship\",\"name\":\"has\",\"target\":\"dtmi:com:microsoft:azure:iot:model{1};1\",\"properties\":[{{\"@type\":\"Property\",\"name\":\"relationshipproperty1\",\"schema\":\"string\"}},{{\"@type\":\"Property\",\"name\":\"relationshipproperty2\",\"schema\":\"string\"}}]}}],\"description\":{{\"en\":\"This is the description of model\"}},\"displayName\":{{\"en\":\"This is the display name\"}},\"@context\":\"dtmi:dtdl:context;2\"}}" + Environment.NewLine;
```

Comment:
Consider naming this "InterfaceTemplate", since that's what you're defining here.

Reply:
This one is a DTDL model template, keeping same name/